### PR TITLE
Operator first support,including ternary,dot etc.

### PR DESCRIPTION
**Pull request by bounceme**
the previous pr is being difficult, sorry

### CODE REVIEW

File: indent/javascript.vim [hunk 1]
```diff
@@ -378,13 +377,31 @@ function GetJavascriptIndent()
...
+      if mnum > 0 && s:Match(mnum, s:operator_first)
+        return indent(mnum)
+      end
+    elseif counts[0] != '1' && counts[1] != '1' && counts[2] != '1'
+      " otherwise, indent 1 level
       return indent(prevline) + s:sw()
-    endif
-  endif
+    end
+    " If previous line starts with a operator...
```
Comment (davidchambers) [inline indent/javascript.vim]: `s/a operator/an operator/`

**Comment by bounceme** [comment]
this is finished, any suggestions for changes?

**Comment by davidchambers** [comment]
I'd guess that you know the indentation code better than any other current contributor, @bounceme. :)

I'm happy to merge this pull request.

**Comment by amadeus** [comment]
@bounceme if interested, we could get you setup as a collaborator? you seem to be the only person capable of pulling out reliable changes to the indent file xD

**Comment by bounceme** [comment]
Why not?I am happy to help

**Pull request merged**